src/site/content/en/blog/how-to-use-local-https/index.md

```diff
@@ -0,0 +1,317 @@
+
+    This create a valid certificate (that will be signed by `mkcert` automatically).
+
+3.  Configure your development server to use HTTPS and the certificate you've created in Step 1.
+4.  ✨ You're done! You can now access `https://{your hostname}` in your browser, without warnings
+
+{% Banner 'caution' %}
+
+Do this only for **development purposes** and **never export or share** the file `rootCA-key.pem` (if you need to know where this file is located to make sure it's safe, run `mkcert -CAROOT`).
+
+{% endBanner %}
+
+{% endDetails %}
+
+## Running your site locally with HTTPS: other options
+
+### Self-signed certificate
+
+You may also decide to not use a local certificate authority like mkcert, and instead **sign your certificate yourself**.
+
+Beware there are a few pitfalls with this approach:
+
+- Browsers don't trust you as a certificate authority and they'll show warnings you'll need to bypass manually. In Chrome, you may use the flag `#allow-insecure-localhost` to bypass this warning automatically on `localhost`. It feels a bit hacky, because it is.
+- Self-signed certificates won't behave in exactly the same way as trusted certificates.
+- This is unsafe if you're working in an insecure network.
+- It's not necessarily easier or faster than using a local CA.
+- If you're not using this technique in a browser context, you may need to disable certificate verification for your server. Omitting to re-enable it in production would be very problematic.
+
+**Note:** [React's](https://create-react-app.dev/docs/using-https-in-development/) and [Vue's](https://cli.vuejs.org/guide/cli-service.html#vue-cli-service-serve) development server HTTPS options create a self-signed certificate under the hood, if you don't specify any certificate. This is quick, but you'll get browser warnings and encounter other pitfalls listed above. Luckily, you can have it all (quick and without pitfalls): as showed in the [mkcert with React example](/how-to-use-local-https/#setup:~:text=With%20a%20React%20development%20server), you can use frontend frameworks' built-in HTTPS option **and** specify a locally trusted certificate created via mkcert or similar.
+
+{% Details %}
+{% DetailsSummary %}
+Why browsers, by default, don't trust self-signed certificates.
+{% endDetailsSummary %}
+
+<figure class="w-figure">
+  <img src="./selfSigned.jpg" alt="Why browsers don't trust self-signed certificates: diagram">
+  <figcaption class="w-figcaption">If you open your locally running site in your browser using HTTPS, your browser will check the certificate of your local development server. Upon seeing that the certificate has been signed by yourself, the browser checks whether you're registered as a trusted certificate authority. Because you're not, your browser can't trust the certificate; it displays a warning telling you that your connection is not secure. You may proceed at your own risk — if you do, an HTTPS connection will be created.</figcaption>
+</figure>
```

Comment:
```suggestion
If you open your locally running site in your browser using HTTPS, your browser will check the certificate of your local development server. When it sees that the certificate has been signed by yourself, it checks whether you're registered as a trusted certificate authority. Because you're not, your browser can't trust the certificate; it displays a warning telling you that your connection is not secure. You may proceed at your own risk—if you do, an HTTPS connection will be created.

<figure class="w-figure">
  <img src="./selfSigned.jpg" alt="Why browsers don't trust self-signed certificates: a diagram.">
  <figcaption class="w-figcaption">Why browsers don't trust self-signed certificates.</figcaption>
</figure>
```

Reply:
Same as above, moving text from caption to body for accessibility.